src/components/illustrations/JumperLogo.tsx

```diff
@@ -66,7 +66,7 @@ export const JumperLogo = () => {
         <style type="text/css">
           {`
             .jumper-logo {
-              width: 248px;
+              width: 175px;
```

Comment:
Good call! We may even decrease it further to something like 155-160ish? Whatever, good one!

Reply:
Seems ok to me, Have to get feedback from the designer if we need to change it I guess